db/migrate/20171208043003_create_families.rb

```diff
@@ -0,0 +1,16 @@
+class CreateFamilies < ActiveRecord::Migration[5.1]
+  def change
+    create_table :families do |t|
+      t.string :first_name
+      t.string :last_name, null: false
+      t.string :address
+      t.string :animals
+      t.integer :capability, null: false
+      t.boolean :prof
+      t.string :preffered_gender
+      t.boolean :bus_driver
+      t.boolean :active
```

Comment:
We probably want to set the default here to be true

Reply:
👍